packages/cw721-soulbound/Cargo.toml

```
@@ -1,21 +1,19 @@
 [package]
-name = "cw721"
-version = "0.14.0"
+name = "cw721-soulbound"
+version = "1.0.1"
 authors = [
+    "Drew Taylor <drew.taylor@philabs.xyz>",
     "Ethan Frey <ethanfrey@users.noreply.github.com>",
     "Orkun Külçe <orkun@deuslabs.fi>",
-    "Drew Taylor <drew.taylor@philabs.xyz>",
 ]
 edition = "2021"
 description = "Definition and types for the CosmWasm-721 NFT interface"
 license = "Apache-2.0"
-repository = "https://github.com/CosmWasm/cw-nfts"
-homepage = "https://cosmwasm.com"
-documentation = "https://docs.cosmwasm.com"
+repository = "https://github.com/archway-network/cw721-soulbound"
 
 [dependencies]
 cw-utils = "0.14"
-cosmwasm-std = { version = "1.0.0" }
+cosmwasm-std = { version = "~1.5.0" }
```

I would suggest using 1.0.0 or bumping if you need any later features because you assure compatibility with every cosmwasm version. Also ~ makes it so only bugfixes can be used, ie: this package is only compatible with >= 1.5.0 && < 1.6.0 

```suggestion
cosmwasm-std = { version = "^1.0.0" }
```

---

okay, fair enough. It's in 2 places btw, since there's 2 packages in this repo